docs/jetson.md

```diff
@@ -0,0 +1,50 @@
+<!--
+# Copyright 2018-2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Triton Inference Server Support for Jetson and JetPack
+
+Triton inference server is officially supported on JetPack starting from JetPack 4.6.  Triton Inference Server on Jetson supports trained AI models from multiple frameworks includings NVIDIA TensorRT, TensorFlow and ONNX Runtime.
+
+On JetPack, although HTTP/REST and GRPC inference protocols are supported, for edge use cases, direct [C API integration](https://github.com/triton-inference-server/server/blob/main/docs/inference_protocols.md#c-api) is recommended, and, hence, Triton Inference Server support is released as a shared library for Jetson.
+
+![Triton on Jetson Diagram](images/triton_on_jetson.png)
+
+Triton Inference Server support on JetPack includes:
+
+* Running models on GPU and DLA
+* Support for multiple frameworks: TensorRT, TensorFlow and ONNX Runtime. PyTorch backend can also be easily added.
```

Comment:
If you are mentioning that PyTorch support can be added it would be useful to point to build instructions for the same.

Reply:
I provided the link to the PyTorch backend (https://github.com/triton-inference-server/pytorch_backend), is it the right one?

Reply:
Yes it is. However we do not officially support the pytorch backend on Jetpack yet and if you wish to add this here you should also add instructions on how to build pytorch + the pytorch backend on Jetson.
I would prefer we do not add the line about Pytorch here unless the above is added in.

Reply:
Understood! Removing it for now.